backend/Controllers/SpeciesController.cs

```diff
@@ -0,0 +1,33 @@
+using Microsoft.AspNetCore.Mvc;
+using WhaleSpotting.Models.Data;
+using WhaleSpotting.Models.Request;
+using WhaleSpotting.Models.Response;
+using WhaleSpotting.Services;
+
+namespace WhaleSpotting.Controllers;
+
+[ApiController]
+[Route("/species")]
+public class SpeciesController : Controller
+{
+    private readonly ISpeciesServices _service;
+
+    public SpeciesController(ISpeciesServices service)
+    {
+        _service = service;
+    }
+
+    [HttpGet("")]
+    public IActionResult GetAllSpecies()
+    {
+        try
+        {
+            List<Species> listOfSpecies = _service.GetAllSpecies();
+            return Ok(new SpeciesResponse { ListOfSpecies = listOfSpecies, Count = listOfSpecies.Count });
```

Comment:
The `SpeciesResponse` should be being returned by the `GetAllSpecies()` method in the service. The controller should only be responsible for returning the response in the API, ideally it shouldn't be where the response is being made

Reply:
Thinking about this more, we may want to have a `GetAllSpeciesResponse` in the service that calls `GetAllSpecies()` and saves the results into a response model?